zwave_js_server/model/value.py

```diff
@@ -11,7 +11,7 @@ def value_id(node: "Node", event: dict) -> str:
     """Return ID of value."""
     command_class = event.get("commandClass")
     endpoint = event.get("endpoint")
-    property_key_name = event.get("propertyKeyName")
+    property_key_name = event.get("propertyKeyName") or event.get("property")
```

Comment:
Can we just always use property? It's important that we stay consistent. 

Do we know why keyName can be None?

Reply:
no we can't as this is for the more complex CC's where one CC can have multiple values, fore example the notification CC

Reply:
I'll try to find an example if you want. Ran into it while testing

Reply:
![Schermafbeelding 2021-01-08 om 10 52 44](https://user-images.githubusercontent.com/6389780/104001231-a4b66980-519f-11eb-8b91-b2603ad43df8.png)

Reply:
It's a sort of (optional) nested structure

Reply:
```suggestion
    property_key_name = event.get("propertyKeyName") or event["property"]
```